network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -725,6 +725,17 @@ def generate_app_install_download_button_field():
     )
 
 
+def generate_bg_cta_link_field():
+    link_value = {
+        "link_to": "external_url",
+        "external_url": fake.url(schemes=["https"]),
+        "label": " ".join(fake.words(nb=2)),
+        "new_window": True,
+    }
```

Comment:
This is beginning to look like a code smell to me, I have created https://mozilla-hub.atlassian.net/browse/TP1-982 in JIRA so we can come up with a helper function for this at a later time!